python/lsst/meas/extensions/psfex/psfex.py

```diff
@@ -796,7 +801,7 @@ def read_samplesLsst(prefs, set, filename, frmin, frmax, ext, next, catindex, co
 
     edgeBit = [k for k, v in getFlags().items() if v == "edge"][0]
 
-    for i, xc, yc in zip(range(nobj), xm, ym):
+    for i, xc, yc in zip(list(range(nobj)), xm, ym):
```

Comment:
Yup, the tests ran fine. I'll retest on Jenkins just to make sure before I merge.